### ISSUE TITLE
https and custom dns documentation or module config

### ISSUE THREAD

**Issue by revpixel** [sheyabernstein/MMM-pihole-stats#8]
Not a big deal as I only had to make minor changes to the MMM-pihole-stats.js file.

Essentially not everyone uses pi.hole to access their site, nor does everyone use HTTP. either add the documentation to the readme or update the module to allow for this. 

The configuration is not a big deal so I'd say changing the README Markdown to add a line regarding the API config is all that's really needed. 

To update the Pi-Hole API URL to support HTTPS change the following line in the MMM-pihole-stats.js file.
From: 
`	apiURL: 'http://pi.hole/admin/api.php',`
To 
`	apiURL: 'https://pi.hole/admin/api.php',`

This module also supports access to the Pi-Hole using custom DNS names. 
`	apiURL: 'https://pihole.customdomain.com/admin/api.php',`

**Comment by Barokai** [comment]
maybe re-adding it to the config would be an idea
https://github.com/sheyabernstein/MMM-pihole-stats/commit/cb2c9861d428f390658f603b11f8cddb1893a115#diff-04c6e90faac2675aa89e2176d2eec7d8L24

**Comment by sheyabernstein** [comment]
Feel free to create a PR, I'll be sure to merge it :)

**Comment by zanix** [comment]
`apiURL` is now a config option

**Comment by Rabbit81** [comment]
Is this possible with a port in the URL?
Installed it and it doesn't work on my mirror.

{
                            module: 'MMM-pihole-stats',
                            position: 'middle_center', // Or any valid MagicMirror position.
                            config: {
                                apiURL:'http://pi.hole:81/admin/api.php',
                                apiToken:'YYYYYYYYa53526a7a5cdb02c3704409f3XXXXXXXXXXXd53b926f702ff',
                                showSources:'true',
                                sourcesCount:'10',
                                showSourceHostnameOnly:'true',
                                updateInterval:'600000',
                                retryDelay:'30000'
                                  // See 'Configuration options' for more information.
                }
                },

**Comment by zanix** [comment]
I don't have time to test this but, I think if you modify `node_helper.js`
https://github.com/sheyabernstein/MMM-pihole-stats/blob/master/node_helper.js#L30

And change
```javascript
request({ url: url, headers: { 'Referer': url }, method: 'GET' }, function (error, response, body) {
```

 To
```javascript
request({ url: url, port: 81, headers: { 'Referer': url }, method: 'GET' }, function (error, response, body) {
```

That might work

**Comment by zanix** [comment]
Created a PR to add the port #15

**Comment by sheyabernstein** [comment]
Resolved by PR #15 - thanks @zanix :)
